src/lib/djerba/util/ini_fields.py

```diff
@@ -6,7 +6,8 @@
 DISCOVERED = 'discovered'
 
 # parameter names
-ARCHIVE_DIR = 'archive_dir'
+ARCHIVE_BASE = 'archive_base'
```

Comment:
More descriptive to call this `ARCHIVE_URL` -- otherwise good work updating the INI fields